libs/iavl/mutable_tree.go

```diff
@@ -32,9 +32,14 @@ func SetProduceDelta(pd bool) {
 	produceDelta = pd
 }
 
+func SetCommitGapOffset(offset int64) {
+	CommitGapOffset = offset
+}
+
 var (
-	ignoreVersionCheck = false
-	produceDelta       = false
+	ignoreVersionCheck       = false
+	produceDelta             = false
+	CommitGapOffset    int64 = 0
```

Comment:
change CommitGapOffset to commitGapOffset because encapsulate SetCommitGapOffset GetCommitGapOffset